### DISCUSSION TITLE
PATH within a custom docker image

### DISCUSSION THREAD

**Discussion by ricktbaker-montera** [actions/actions-runner-controller#1518] [Questions]
Greetings.  Have everything up and running, now working on a custom docker image for the actual runner where I'd like to have things like nodenv installed.    I can't seem to find a way to override the PATH that is being set within /etc/environment.   Added into various places and regardless it continues to be set to: /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/home/runner/.local/bin

Read through the docs, and don't see anything on how we should add to the PATH for the runner user within a custom docker image.  Maybe I'm just missing something?

**Comment by ricktbaker-montera** [comment]
Seems to be that for now, I can just add to PATH in /etc/environment in my custom dockerfile.  Unsure if there is a better way that might exist already however.

**Reply by sondrelg** [reply]
How are you adding to PATH in /etc/environment @ricktbaker-montera?

**Comment by LalitLab** [comment]
Hi @ricktbaker-montera 
Could you share the steps or your docker file to create a custom image for the runner.
I am trying to pre-install sonarqube cli on the runner image but unable to make it work.
Thank you
